Library/Formula/bip.rb

```diff
@@ -1,15 +1,31 @@
-require "formula"
-
 class Bip < Formula
-  homepage "http://bip.milkypond.org"
-  url "http://ftp.de.debian.org/debian/pool/main/b/bip/bip_0.8.9.orig.tar.gz"
+  # Auto redirects to https but currently has broken/untrusted warning.
+  # homepage "https://bip.milkypond.org"
```

Comment:
I'm open to leaving it, but I tend to feel redirecting people to a homepage that Chrome will automatically block with no option to advance, and Firefox will fire great big red warnings about being insecure isn't a great user experience :crying_cat_face: